<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentDialog</class>
 <widget class="QDialog" name="PaymentDialog">
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Оплата чека</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QFrame" name="toPayFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="toPayLayout">
      <item>
       <widget class="QLabel" name="toPayLabel">
        <property name="text">
         <string>К оплате: 0.00 ₽</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="paymentsTable">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="columnCount">
      <number>2</number>
     </attribute>
     <column>
      <property name="text">
       <string>Способ оплаты</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сумма, ₽</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btnLayout">
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>+ Добавить способ оплаты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBtn">
       <property name="text">
        <string>Удалить строку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fullAmountBtn">
       <property name="text">
        <string>Вся сумма в одну строку</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="totalLabel">
     <property name="text">
      <string>Итого оплат: 0.00 ₽</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="diffLabel">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
